hack/make/.integration-daemon-start

```diff
@@ -16,6 +16,10 @@ if [ "$(go env GOOS)" = 'windows' ]; then
        return
 fi
 
+if [ ! -x "/etc/docker/seccomp.json" ]; then
+	install_seccomp "$base/../../profiles/seccomp/default.json"
```

Comment:
> Couldn't/shouldn't the default profile path be a daemon flag/config option? (esp. so that the tests could point directly to the copy in the repo)

There has been some discussion around this in #24221 and that was how this feature was initially proposed. This PR basically follows this specific comment https://github.com/docker/docker/pull/24221#issuecomment-231457809 and so, I think it isn't really ok to have a daemon flag/config. I'm still open to have this though. /cc @justincormack